### DISCUSSION TITLE
Beyond Images: Other Media

### DISCUSSION THREAD

**Discussion by sirinath** [GraphiteEditor/Graphite#1463] [Ideas/Feedback]
Can the editing experience (i.e. "node graph compositing engine that fuses layers with nodes, providing a fully nondestructive editing experience") be extended to audio, video and live audio and video.

**Comment by Keavon** [comment]
This is a long-term aspiration, yes, but not a current focus.

**Reply by sirinath** [reply]
Great to hear. Maybe engine can be factored so that other functionality can be easily added and build upon.

**Reply by Keavon** [reply]
That's indeed our approach.
